src/routes/search/+page.svelte

```diff
@@ -0,0 +1,111 @@
+      searchFunction(helper) {
+        const searchUI = document.querySelectorAll(".stats, .hits, .pagination");
+
+        searchUI.forEach((element) => {
+          (element as HTMLElement).style.display = helper.state.query === "" ? "none" : "";
+        });
+
+        helper.search();
+      },
+    });
+
+    search.addWidgets([
+      configure({
+        hitsPerPage: 3,
+        attributesToSnippet: ["content:14"],
+        snippetEllipsisText: " [...]",
+      }),
+      hits({
+        container: ".hits",
+        templates: {
+          empty: "No results found.",
+          item(hit) {
+            return searchHitsTemplate(hit);
+          },
+        },
+      }),
+      searchBox({
+        container: ".searchbox",
+        placeholder: "Search LDAF",
+        autofocus: false,
+        showReset: false,
+        cssClasses: {
+          // root: // the root element of the widget.
+          form: "usa-search", // the form element.
+          input: "usa-input", // the input element.
+          // reset: // the reset button element.
+          // resetIcon: // the reset button icon.
+          // loadingIndicator: // the loading indicator element.
+          // loadingIcon: // the loading indicator icon.
+          submit: "usa-button usa-search__submit", // the submit button element.
+          submitIcon: "usa-search__submit-icon", // the submit button icon.
+        },
+      }),
+      stats({
+        container: ".stats",
+        templates: {
+          text(data, { html }) {
+            let count = "";
+
+            if (data.hasManyResults) {
+              count += `${data.nbHits} results`;
+            } else if (data.hasOneResult) {
+              count += `1 result`;
+            } else {
+              count += `No result`;
+            }
+
+            return html`<strong>${count}${data.query ? ` for "${data.query}"` : ""}</strong>`;
+          },
+        },
+      }),
+      pagination({
+        container: ".pagination",
+        showFirst: true,
+        showLast: true,
+        cssClasses: {
+          root: "usa-pagination", // the root element of the widget.
+          noRefinementRoot: "", // the root container without results.
+          list: "usa-pagination__list", // the list of results.
+          item: "", // the item in the list of results.
+          lastPageItem: "", // the last item.
+          previousPageItem: "usa-pagination__link-text usa-pagination__previous-page", // the previous item.
+          nextPageItem: "usa-pagination__link-text usa-pagination__next-page", // the next item.
+          pageItem: "usa-pagination__item usa-pagination__page-no", // the page items.
+          selectedItem: "usa-current", // the selected item.
+          disabledItem: "", // the disabled item.
+          link: "usa-pagination__button", // the link elements.
+        },
+        templates: {
+          next: "Next",
+          previous: "Previous",
+        },
+      }),
+    ]);
+
+    search.start();
```

Comment:
so that method name is a little misleading and is just initializing the whole `instantsearch` apparatus and not necessarily making a query. The search function that wraps around queries is [here](https://github.com/la-ldaf/ldaf-site/pull/212/files#diff-4fd57f831561a3f8ff467da7008ad85aee43203174f0bde4e9a5c9227f9f2153R18-R31) and won't make a request if there's no query. The default just gives you whatever's first: 
> If no query parameter is set, the textual search will match with all the objects.

Reply:
I think we should definitely consider debouncing in the future if we see a need for it, but the default Algolia behavior should be fine for now.